openedx/features/idea/views.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# Create your views here.
```

Comment:
Let's please remove this comment as it's not serving any purpose. OR correct me if I am wrong?